lib/cc/cli/command.rb

```diff
@@ -24,7 +24,7 @@ def execute
         run
       end
 
-      def say(message)
+      def say(message = "")
```

Comment:
You're only using this in one place, and I personally find the idea of calling `say` without params a bit surprising when reading the calling code. Is this really better than being explicit at the call site?